app/src/main/java/edu/agh/susgame/front/model/graph/Node.kt

```diff
@@ -2,32 +2,53 @@ package edu.agh.susgame.front.model.graph
 
 import edu.agh.susgame.front.util.Coordinates
 
-open class Node(
+abstract class Node(
     val id: Int,
```

Comment:
Jakby Ci się tylko chciało to zachęcam, żeby to zmienić na:
```
data class NodeId(val value: Int)

abstract class Node(
    val id: NodeId,
    // ...
```

Reply:
zmieniłem, jeszcze pociągnąłem zmianę dla edge (`EdgeId`) dla spójności